### ISSUE TITLE
problem when using uvloop

### ISSUE THREAD

**Issue by shaljam** [vxgmichel/aioconsole#5]
I'm getting the following exception when using `uvloop`:
```
Traceback (most recent call last):
  File "main.py", line 353, in handle_console
    line = await ainput(">>> ")
  File "/Users/ali/.pyenv/versions/3.5.3/lib/python3.5/site-packages/aioconsole/stream.py", line 147, in ainput
    reader, writer = yield from get_standard_streams(loop=loop)
  File "/Users/ali/.pyenv/versions/3.5.3/lib/python3.5/site-packages/aioconsole/stream.py", line 135, in get_standard_streams
    cache[key] = yield from connection
  File "/Users/ali/.pyenv/versions/3.5.3/lib/python3.5/site-packages/aioconsole/stream.py", line 123, in create_standard_streams
    reader, writer = yield from future
  File "/Users/ali/.pyenv/versions/3.5.3/lib/python3.5/site-packages/aioconsole/stream.py", line 109, in open_pipe_connection
    loop.remove_reader(transport._fileno)
AttributeError: 'uvloop.loop.WriteUnixTransport' object has no attribute '_fileno'
```

**Comment by vxgmichel** [comment]
Thanks for the report!

This issue has been fixed in 0dbc3b0 (PR #2). It will become available with v0.1.3, that I will release soon.

**Comment by vxgmichel** [comment]
[v0.1.3](https://github.com/vxgmichel/aioconsole/releases/tag/v0.1.3) is now released!
